metrics/android-render-test-runner/render-tests/symbol-placement/line-center-buffer-tile-map-mode/metrics.json

```diff
@@ -15,20 +15,20 @@
         [
             "probeGFX - default - end",
             10,
-            33,
-            62,
+            22,
+            32,
             4,
             [
-                321262,
-                321262
+                158509,
+                158509
```

Comment:
Interesting to see the peek memory lowering.

Reply:
it is expected and it actually verifies, that the intended functionality works - only memory, required for the particular tile rendering is being allocated.

Reply:
Looks like we store metrics only from a last render. Since we kill all buckets between renders, there are less buffer to be uploaded.

Reply:
> Looks like we store metrics only from a last render.

Not really. Before render orchestrator was keeping data from all `render()` calls, i.e. keeping all tiles |=> memory peak increased. Now unneeded memory gets freed |=> memory peak decreases

Reply:
@pozdnyakov yep 👍, just checked and stats are appended `result.stats += resultForTile.stats;`